api/controllers/LogoutController.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Why create a separate controller? All of the user's actions can go in UserController